src/main/java/pl/prawko/prawko_server/controller/ExceptionController.java

```diff
@@ -23,4 +29,43 @@ public ApiResponse handleWrongFileType(final MultipartException exception) {
         return new ApiResponse(exception.getMessage());
     }
 
+    /**
+     * Handles cases when entity already exists.
```

Comment:
remwmbwe about articles: "the entity". Check every place for a missing "a"/"an"/"the"